core/chain-configs/src/genesis_config.rs

```diff
@@ -3,11 +3,13 @@
 //! NOTE: chain-configs is not the best place for `GenesisConfig` since it
 //! contains `RuntimeConfig`, but we keep it here for now until we figure
 //! out the better place.
+use anyhow::Context;
```

Comment:
Following the already existing style, this import should be in the block below :neutral_face: